Base/usr/share/man/man1/checksum.md

```diff
@@ -0,0 +1,22 @@
+## Name
+
+checksum - Helper program for calculating checksums
```

Comment:
Most programs use lower case in this description. Like: "program - does something" not "program - Does something"